packages/uix-core/src/promises/timed.ts

```diff
@@ -12,21 +12,29 @@
  * @internal
  */
 export function timeoutPromise<T>(
-  description: string,
+  describe: string | (() => string),
```

Comment:
Because generating an error message can involve expensive serializations, the API for describing the operation for errors now lazy evaluates.